qualang_tools/utils/qua_utils.py

```diff
@@ -1,6 +1,77 @@
 import numpy as np

```

Comment:
This is without any logic. The idea is to make sure that the result is the same as numpy `arange`, which currently does not happen...
Cases where this would break:
a_min = 0.1, a_max = 1.0, da = 0.2 - Because +da/2 would give us 1.1, and it **might** be included...
a_min = 10, a_max = 20, da = 1 - Because it is an integer.

There are many more examples, and some are listed on the confluence page:
https://quantum-machines.atlassian.net/wiki/spaces/PLAN/pages/1144389988/Improve+for+behavior

This is a very big effort, I suggest to move it to a separate PR